clients/include/blas_ex/testing_gemm_ex3.hpp

```diff
@@ -1947,12 +1963,6 @@ void testing_gemm_ex3(const Arguments& arg)
     // check for invalid sizes
     bool invalid_size = M < 0 || N < 0 || K < 0 || lda < A_row || ldb < B_row || ldc < M || ldd < M;
 
-    // size checking is only needed for int8x4
-    bool pack_to_int8x4 = arg.flags & rocblas_gemm_flags_pack_int8x4;
-    // bool int8_invalid   = (pack_to_int8x4 && std::is_same<Ti, int8_t>{}
-    //                      && (K % 4 != 0 || (transA != rocblas_operation_none && lda % 4 != 0)
-    //                          || (transB == rocblas_operation_none && ldb % 4 != 0)));
-
```

Comment:
What is this change?    We are seeing navi3x int8 failures is it unrelated work?